test/model/tools/model_verify_debug_log/main.cpp

```diff
@@ -88,8 +90,11 @@ main(int argc, char *argv[])
      * Parse the command-line arguments.
      */
     __wt_optwt = 1;
-    while ((ch = __wt_getopt(progname, argc, argv, "c:h:j:?")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "C:c:h:j:?")) != EOF)
         switch (ch) {
+        case 'C':
+            conn_config = __wt_optarg;
```

Comment:
Should this be a wholesale override, or would we be better off appending it? That way the user can keep the defaults and just change what they need. (My understanding is that they would still be able to override the defaults as later options override earlier.)

Reply:
Ah, that's a much better idea! Thank you.